### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by mklomp**
This PR has:

- cleanup and os-independant paths using pathlib
- woodlike colors
- exporting step files as well

### CODE REVIEW

File: bill_of_materials.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+This repositoay contains the frame for the MIRTE Light, Basic and Pioneer.
+The releases will include files for lasercutting (DXF), 3D printing (STL),
+and files to generate renders for workshops (STEP). 
+
```
Comment (ArendJan) [inline bill_of_materials.md:5]: Maybe add the current thickness (4mm) somewhere in this file

File: README.md [hunk 2]
```diff
@@ -1 +1,3 @@
-# Mirte_Frame
\ No newline at end of file
+# Mirte_Frame
+
+Please see the MIRTE docs.
```
Comment (ArendJan) [inline README.md]: Maybe add a link somewhere

**Review by ArendJan — APPROVED** [review]
Looks good! Why are the FCStd files changed?

File: bill_of_materials.md (hunk 1 above)
Comment (mklomp) [reply bill_of_materials.md:5]: Currently the default is 3mm.

File: README.md (hunk 2 above)
Comment (mklomp) [reply README.md]: There is nothing there either about the frame at the moment. So changed to TODO.

**Comment by ArendJan** [comment]
> Looks good! Why are the FCStd files changed?

Change of material color

**Pull request merged**